src/test/optimizer/strategy/in_expression_rewrite_rule_test.cpp

```diff
@@ -248,19 +253,25 @@ TEST_F(InExpressionRewriteRuleTest, JoinStrategy) {
 TEST_F(InExpressionRewriteRuleTest, AutoStrategy) {
   auto rule = std::make_shared<InExpressionRewriteRule>();
 
+  const auto cardinality_estimator = CardinalityEstimator{};
+
   {
     // Disjunction for single element
     const auto input_lqp = PredicateNode::make(single_element_in_expression, node);
     const auto result_lqp = StrategyBaseTest::apply_rule(rule, input_lqp);
     const auto expected_lqp = PredicateNode::make(equals_(col_a, 1), node);
     EXPECT_LQP_EQ(result_lqp, expected_lqp);
+
+    EXPECT_FLOAT_EQ(cardinality_estimator.estimate_cardinality(result_lqp), 1000.f / 200 * 1);
   }
 
   {
     // ExpressionEvaluator for five elements
     const auto input_lqp = PredicateNode::make(five_element_in_expression, node);
     const auto result_lqp = StrategyBaseTest::apply_rule(rule, input_lqp);
     EXPECT_EQ(result_lqp, input_lqp);
+
+    // No cardinality check here, as unmodified IN expressions are currently not estimated by the CardinalityEstimator.
```

Comment:
Do I have to read the rule to understand why 5 elements leads to an "unmodified" expression? I'd guess the threshold for a table is larger? Maybe comment?

Reply:
Yes, rewrites to join only happen with at least `MIN_ELEMENTS_FOR_JOIN` elements. Someone who looks into this rule and its test (and not at a small PR that does not really touch the rule) should have already seen that a couple of times, so I am not sure if a comment is helpful.

Reply:
Hmm, since both the 100 and 5 are so arbitrary, I think a one liner that the 5 is below `MIN_ELEMENTS_FOR_JOIN` and 100 is above would help.

Reply:
better?